website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -302,62 +302,68 @@ resource "newrelic_cloud_aws_integrations" "bar" {
 ```
 ## Argument Reference
 
```

Comment:
Note for the reviewer:
The following changes (made to this file - the AWS Cloud Integrations docs page) have been made as part of the [second commit](https://github.com/newrelic/terraform-provider-newrelic/pull/2399/commits/3537858d0c95df8dc93c3344e70d58bcbe6bafb7) in this PR as a proposal for changes in this page, to reformat the contents of this page given there's a large number of AWS services supported by this integration (51).